docs/en-US/guide/ssr.md

```diff
@@ -0,0 +1,184 @@
+---
+title: SSR
+lang: en-US
+---
+
+# Server-Side Rendering (SSR)
+
+You need to carry out special handling during SSR to avoid hydrate errors.
+
+## Provide an ID
+
+You need to inject the same ID value into the server side and client side.
+
+### Native SSR
+
+```ts
+// src/main.js (irrelevant code omitted)
+import { createApp } from 'vue'
+import { ID_INJECTION_KEY } from 'element-plus'
+import App from './App.vue'
+
+const app = createApp(App)
+app.provide(ID_INJECTION_KEY, {
+  prefix: 1024,
+  current: 0,
+})
+```
+
+### Nuxt
+
+```vue
+// app.vue (irrelevant code omitted)
+<script setup>
+import { ID_INJECTION_KEY } from 'element-plus'
+
+provide(ID_INJECTION_KEY, {
+  prefix: 1024,
+  current: 0,
+})
+</script>
+```
+
+## Teleports
+
+[Teleport](https://vuejs.org/guide/scaling-up/ssr.html#teleports) is used internally by multiple components in Element Plus (eg. ElDialog, ElDrawer, ElTooltip, ElDropdown, ElSelect, ElDatePicker ...), so special handling is required during SSR.
```

Comment:
```suggestion
[Teleport](https://vuejs.org/guide/scaling-up/ssr.html#teleports) is used internally by multiple components in Element Plus (eg. ElDialog, ElDrawer, ElTooltip, ElDropdown, ElSelect, ElDatePicker,ElAutocomplete, ElCascader, ElColorPicker, ElPopconfirm, ElPopover, ElSlider, ElTable, ElTimePicker, ElTimePicker), so special handling is required during SSR.
```

Reply:
I do not think it is necessary to list them all here.